skills/gh-address-comments/scripts/_bd_utils.py

```
@@ -0,0 +1,78 @@
+All bead-related scripts import from here so the logic stays in one place.
+"""
+
+from __future__ import annotations
+
+import os
+import subprocess
+import sys
+
+
+def _bd_in_path() -> bool:
+    return subprocess.run(["which", "bd"], capture_output=True).returncode == 0
```

Ruff (0.15.10)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)